.circleci/README.asciidoc

```diff
@@ -0,0 +1,70 @@
+== openQA circleci workflow
+
+=== Goal 
+
+Provide a way to run tests with pre-approved list of dependencies both in CI and locally
+
+=== Dependency artefacts
+
+- dependencies.txt list of dependencies to test against.
+- autoinst.sha contains sha of os-autoinst commit for integration testing.
+  When value is empty, the testing will run against latest master
+
+=== Managing and troubleshooting dependencies
+
+dependencies.txt and autoinst.sha are normally managed by dedicated bot, 
+defined in 'nightly' job in config.cnf. The bot manage master branch only;
+and to is possible to add another branch to config.cnf, so daily PR will be 
```

Comment:
I don't understand "and to is possible". Do you mean "Also, it is possible …"?

Reply:
I don't understand it, too. And the beginning of the sentence is grammatically better written as "The bot manages only the master branch.". The rest should be an own sentence.